src/Http/Http/src/BindingAddress.cs

```diff
@@ -88,7 +89,15 @@ public static BindingAddress Parse(string address)
             }
             else
             {
-                pathDelimiterStart = address.IndexOf(":", schemeDelimiterEnd + UnixPipeHostPrefix.Length, StringComparison.Ordinal);
+                if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+                {
+                    pathDelimiterStart = address.IndexOf(":", schemeDelimiterEnd + UnixPipeHostPrefix.Length + 2, StringComparison.Ordinal);
```

Comment:
Add a comment explaining the + 2.

Also it might be a bit cleaner like this:

```C#
var unixPipeHostPrefixLength = UnixPipeHostPrefix.Length;
if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
{
   // Windows has drive letters (c:)
   unixPipeHostPrefixLength  += 2;
}

pathDelimiterStart = address.IndexOf(":", schemeDelimiterEnd + unixPipeHostPrefixLength , StringComparison.Ordinal);
```